docs/guides/continuous-integration/aws-codebuild.mdx

```diff
@@ -434,21 +435,22 @@ are leveraging three useful features of
 1. [Recording test results with the `--record` flag](https://on.cypress.io/how-do-i-record-runs)
    to [Cypress Cloud](https://on.cypress.io/cloud):
 
-   - In-depth and shareable [test reports](/guides/cloud/runs).
+   - In-depth and shareable
+     [test reports](/guides/cloud/recorded-runs#Latest-Runs).
```

Comment:
is this valid?

Reply:
The 'sharable' part? You can download a .csv from the analytics sections. I'll add a ticket to revise this.